cpp/src/bindings.cpp

```diff
@@ -52,6 +52,10 @@ PYBIND11_MODULE(_pypowsybl, m) {
 
     m.def("create_four_substations_node_breaker_network", &pypowsybl::createFourSubstationsNodeBreakerNetwork, "Create an 4-substation example network");
 
+    m.def("create_battery_network", &pypowsybl::createBatteryNetwork, "Create an example network with batteries");
```

Comment:
check this https://github.com/powsybl/pypowsybl/pull/114
preferred use import xml in tests

Reply:
Thanks @yichen88 :
just to log what has been done : Etienne has left the creation methods, but "hidden" from the normal user using a leading underscore.

It allows to use those tests networks from unit test while not exposing them directly to the user, which was the initial goal I think : @geofjamg we can change it in the future if really needed.